vonage/assets/logs/vonage_tests.yaml

```diff
@@ -0,0 +1,94 @@
+      "dcs" : "0",
+      "country_name" : "India",
+      "from" : "918050290593",
+      "currency" : "EUR",
+      "date_received" : "2024-11-05T07:02:52+00:00",
+      "id" : "19118688-07e9-45b8-8ca4-340850f8dde7",
+      "direction" : "outbound",
+      "account_ref" : "",
+      "total_price" : "0.07268000",
+      "client_ref" : "1|19118688-07e9-45b8-8ca4-340850f8dde7|1730790172031|eu-central-1",
+      "validity_period" : "241106080252000+",
+      "network_name" : "BSNL - Karnataka",
+      "message_id" : "19118688-07e9-45b8-8ca4-340850f8dde7",
+      "ip_address" : "49.249.95.150",
+      "account_id" : "bf8d8cc9",
+      "udh" : "",
+      "forced_from" : "918050290593",
+      "error_code" : "0",
+      "to" : "919481686682",
+      "status" : "delivered"
+    }
+  result:
+    custom:
+      account_id: "bf8d8cc9"
+      account_ref: ""
+      changed_from: ""
+      client_ref: "1|19118688-07e9-45b8-8ca4-340850f8dde7|1730790172031|eu-central-1"
+      country: "IN"
+      country_name: "India"
+      currency: "EUR"
+      date_finalized: "2024-11-05T07:18:37+00:00"
+      date_received: "2024-11-05T07:02:52+00:00"
+      dcs: "0"
+      direction: "outbound"
+      error_code: "0"
+      error_code_description: "Delivered"
+      forced_from: "918050290593"
+      from: "918050290593"
+      id: "19118688-07e9-45b8-8ca4-340850f8dde7"
+      ip_address: "49.249.95.150"
+      latency: "945284"
+      message_body: "Hello from Vonage, checking for new data!"
+      message_id: "19118688-07e9-45b8-8ca4-340850f8dde7"
+      network: "40471"
+      network_name: "BSNL - Karnataka"
+      status: "delivered"
+      to: "919481686682"
+      total_price: "0.07268000"
+      udh: ""
+      validity_period: "241106080252000+"
+    message: |-
+      {
+        "country" : "IN",
+        "changed_from" : "",
+        "latency" : "945284",
+        "date_finalized" : "2024-11-05T07:18:37+00:00",
+        "network" : "40471",
+        "error_code_description" : "Delivered",
+        "message_body" : "Hello from Vonage, checking for new data!",
+        "dcs" : "0",
+        "country_name" : "India",
+        "from" : "918050290593",
+        "currency" : "EUR",
+        "date_received" : "2024-11-05T07:02:52+00:00",
+        "id" : "19118688-07e9-45b8-8ca4-340850f8dde7",
+        "direction" : "outbound",
+        "account_ref" : "",
+        "total_price" : "0.07268000",
+        "client_ref" : "1|19118688-07e9-45b8-8ca4-340850f8dde7|1730790172031|eu-central-1",
+        "validity_period" : "241106080252000+",
+        "network_name" : "BSNL - Karnataka",
+        "message_id" : "19118688-07e9-45b8-8ca4-340850f8dde7",
+        "ip_address" : "49.249.95.150",
+        "account_id" : "bf8d8cc9",
+        "udh" : "",
+        "forced_from" : "918050290593",
+        "error_code" : "0",
+        "to" : "919481686682",
+        "status" : "delivered"
+      }
+    tags:
+     - "source:vonage"
```

Comment:
This line should be 
`     - "source:LOGS_SOURCE"` 
based on the example

Reply:
Resolved, fixed in previous commits.